client/src/redux/sound-mode-saga.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable require-yield */
+
+import { takeEvery } from 'redux-saga/effects';
+import store from 'store';
+
+const soundKey = 'fcc-sound';
+
+export function setSound(setting) {
+  store.set(soundKey, setting);
+}
+
+function* updateLocalSoundSaga({ payload: { user, sound } }) {
+  if (user) {
+    const { sound = false } = user;
+    return setSound(sound);
+  }
+  return setSound(sound);
```

Comment:
```suggestion
  if (user) {
    const { sound = false } = user;
    setSound(sound);
  } else if (typeof sound !== 'undefined') {
    setSound(sound);
  }
```
This should stop the local storage weirdness @ShaunSHamilton spotted.  Similar changes would need applying to `updateLocalThemeSaga` otherwise it will get clobbered by the `updateUserFlagComplete` events, too.